packages/ai-bot/lib/send-response.ts

```diff
@@ -106,17 +109,19 @@ export class Responder {
 
   async handleFunctionToolCalls(msg: {
     role: string;
+    content?: string | null;
     tool_calls?: ChatCompletionMessageToolCall[];
   }) {
     for (const toolCall of msg.tool_calls || []) {
       log.debug('[Room Timeline] Function call', toolCall);
       try {
-        let optionPromise = sendOption(
-          this.client,
-          this.roomId,
-          this.deserializeToolCall(toolCall),
-          this.initialMessageReplaced ? undefined : this.initialMessageId,
-        );
+        let optionPromise = sendOption({
+          client: this.client,
+          roomId: this.roomId,
+          content: msg.content,
+          functionCall: this.deserializeToolCall(toolCall),
+          eventToUpdate: this.initialMessageId,
```

Comment:
If there is more than one tool call, does this still work? It looks like it might replace the initial event with each tool call resulting in just one at the end

Reply:
It seems we need to update our message structure. Currently, a message can have only one command, but we need it to support multiple commands.